storybook/src/elements/nuxeo-video/nuxeo-video-conversions.stories.js

```diff
@@ -0,0 +1,28 @@
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import { text } from '@storybook/addon-knobs';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-video/nuxeo-video-conversions';
+import DocumentBuilder from '../../data/documents.data';
+import videoProperties from '../../video/videoProperties.js';
+
+storiesOf('UI/nuxeo-video', module).add('Nuxeo Video Converter', () => {
+  const documentBuilder = new DocumentBuilder()
+    .setType('File')
+    .setProperties({ 'vid:transcodedVideos': videoProperties['vid:transcodedVideos'] });
+  documentBuilder.build();
+  const label = text('Label', 'Video Conversion');
+  return html`
+    <style>
+      *[role='widget'] {
```

Comment:
We shouldn't need (and rely) on this `role="widget"` here. It is something generated by Studio for layouts.